compiler/rustc_mir/src/util/borrowck_errors.rs

```diff
@@ -68,9 +68,10 @@ impl<'cx, 'tcx> crate::borrow_check::MirBorrowckCtxt<'cx, 'tcx> {
             err.span_label(
                 new_loan_span,
                 format!(
-                    "mutable borrow starts here in previous \
-                     iteration of loop{}",
-                    opt_via
+                    "{}{} was already mutably borrowed in the previous iteration of the loop{}",
```

Comment:
I realized that the error I suggested was not ideal. Would you be open to this instead? Sorry I keep asking you to change it! I just want to make sure the error is as clear as possible :)

```suggestion
                    "{}{} was mutably borrowed here in the previous iteration of the loop{}",
```